# **Statistics Assignment 2**

In [1]:
# Importing libraries 
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
from scipy import stats
%matplotlib inline

# Problem Statement 1:

In each of the following situations, state whether it is a correctly stated hypothesis

testing problem and why?

- 1.H0: μ = 25, H1: μ ≠ 25
- 2.H0: σ > 10, H1: σ = 10
- 3.H0: x = 50, H1: x ≠ 50
- 4.H0: p = 0.1, H1: p = 0.5
- 5.H0: s = 30, H1: s > 30

``Answers``
- 1. Yes, it is Correctly Stated.
  It is a two tailed hypothesis (Whenever , your alternate hypothesis does not equals to something then your calculated z-values can be on left side or right side so in that scenerio you have to use two-tailed test )

- 2. Incorrect Statement. Firstly, null and alternate hypothesis combined aren't completing the entire sample space and alternate hypothesis is not complimemt of the null hypothesis. Also, equality is mostly stated in null hypothesis.

- 3. Yes, it is Correctly Stated.
  It is a two tailed hypothesis (Whenever , your alternate hypothesis does not equals to something then your calculated z-values can be on left side or right side so in that scenerio you have to use two-tailed test )
  

  Eg:
  ``Note``:x is with bar or x-bar
  - the null hypothesis cannot be stated in terms of statistic, x̄.
  - Hence, in this case it ' s not correctly stated.

  -  4. In this case we can see our probability of null hypothesis value is not same as alternate hypothesis value although alternate one contains equality sign , Still it 's not proper hypothesis.

  Hence, it is not correctly stated.
   
- 5. As same as 2nd point that is , Incorrect Statement. Firstly, null and alternate hypothesis combined aren't completing the entire sample space and alternate hypothesis is not complimemt of the null hypothesis.


# **Problem Statement 2**:

The college bookstore tells prospective students that the average cost of its textbooks is Rs. 52 with a standard deviation of Rs. 4.50.

 A group of smart statistics students thinks that the average cost is higher.
 
  To test the bookstore’s claim against their alternative, the students will select a random sample of size 100.
  
   Assume that the mean from their random sample is Rs. 52.80. Perform a hypothesis test at the 5% level of significance and state your decision.

In [13]:
# Given:
# mu (population mean): 52
# std = 4.50
# sample size (randomly selected): 100
# x-bar (sample mean): 52.80
# significance value = 5% or 0.05

# Doing Hypothesis Testing:
# claims made by book seller,
# First Step:
# h0: average cost is not higher(mean = 52) , h1: average cost is  higher(mean! = 52)
# so, this is Two-tailed test as there is a equality sign in alternate hypothesis which is
#complement of null hypothesis.

#Second Step:
#Doing Z-Test:
# z = (x_mean (sample mean) - population_mean(mu) / Standard Error)

# To find Standard Error : (std / under_root of n(sample size))
# (4.50 / under_root of 100) i.e., (4.50/10) = 0.45

# z = 52.80 - 52 / 0.45
# z = 1.77

#Third Step:
#Z of significance level
# as this two-tailed we have to do half of significance value(See Figure - 1 Below )
# that becomes , 0.025 both sides of the distribution 
# Z(0.05) = +- 1.96(See Figure - 2) 
# So , our critical region is from -1.96 to +1.96.
# here my point that is 1.77 not falling in crtical region 
# hence , we have to accept our null-hypothesis and reject alternate hypothesis.
# Conclusion: Average cost that is claimed by book seller is true or correctly stated.
              #x_bar = 52

#Code:
standard_error = 4.50/np.sqrt(100)
z = (52.80 - 52) / standard_error  
z
print("critical region is +1.96 to -1.96")
print("As are z - value not lie in critical region we have to accept null_hypo...")
print('As claimed made by college book store is correct: x = 52 ')

critical region is +1.96 to -1.96
As are z - value not lie in critical region we have to accept null_hypo...
As claimed made by college book store is correct: x = 52 


**Figure - 1**

![Alt text](../../../../../c:/Users/amitt/OneDrive/Desktop/WhatsApp%20Image%202022-11-20%20at%2006.23.28.jpg)

**Figure - 2**

![Figure - 2](../../../../../c:/Users/amitt/OneDrive/Desktop/Screenshot%202022-11-20%20062947.png)

# **Problem Statement 3**:

A certain chemical pollutant in the Genesee River has been constant for several years with mean μ = 34 ppm (parts per million) and standard deviation σ = 8 ppm.

 A group of factory representatives whose companies discharge liquids into the river is now claiming that they have lowered the average with improved filtration devices.
 
  A group of environmentalists will test to see if this is true at the 1% level of significance. Assume that their sample of size 50 gives a mean of 32.5 ppm.


Perform a hypothesis test at the 1% level of significance and state your decision.

In [19]:
#Code To Do Hypothesis Testing

# H0 : mean <= 34 right-tailed-test
# H1 : mean > 34

#standard_error = standard_deviation / np.sqrt(sample_size)
standard_error = 8 / np.sqrt(50)

#z = (sample_mean - population_mean)/standard_error
z_statistics = (32.5 - 34)/standard_error
z_statistics
# checking crtical region with respect to significance level that we have calculated
critical_value = 2.3
#from above
print("Yes,claim made by factory representatives is correct , average has decreased from 34ppm")
print("Our z_statistics value not lie within crtical value 2.3")
print("we have to accept null-hypothesis and rejected alternate hypothesis")

Yes,claim made by factory representatives is correct , average has decreased from 34ppm
Our z_statistics value not lie within crtical value 2.3
we have to accept null-hypothesis and rejected alternate hypothesis


# **Problem Statement 4**:

Based on population figures and other general information on the U.S. population, suppose it has been estimated that, on average, a family of four in the U.S. 

spends about $1135 annually on dental expenditures. Suppose further that a regional dental association wants to test to determine if this figure is accurate for their area of country.

 To test this, 22 families of 4 are randomly selected from the population in that area of the country and a log is kept of the family’s dental expenditure for one year. The resulting data are given below.
 
  Assuming, that dental expenditure is normally distributed in the population, use the data and an alpha of 0.5 to test the dental association’s hypothesis.


1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994

In [6]:
data = [1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994]

sample_mean = np.mean(data)
sample_std = np.std(data)

# As the sample size is small and population std deviation is unknown, we will use t-test

# H0 : mean = 1135
# H1 : mean != 1135

std_err = sample_std / np.sqrt(22)  # standard error = std deviation / (sample size)**0.5
t_statistic = (sample_mean - 1135)/ std_err # z_statistic = (Sample mean - Population mean) / standard error
print(t_statistic)
#df:degree of freedom = sample - 1 => 22-1 => 21
# As this is a two tailed test with alpha = 0.5 and df = 21. Using, t table, we get t_critical = -0.686 and +0.686.

#Our t_statistic = -2.07 which lies in the rejection region. So, we have to reject the null hypothesis.

print("So, on average a family of four in the U.S. doesn't spends about $1135 annually on dental expenditure")

-2.070747228595759
So, on average a family of four in the U.S. doesn't spends about $1135 annually on dental expenditure


# figure ``t-statistics``
![Alt text](ts.png)

# **Problem Statement 6:**

Suppose that in past years the average price per square foot for warehouses
in the United States has been 32.28 dollars.

A national real estate investor
wants to determine whether that figure has changed now. The investor hires
a researcher who randomly samples 19 warehouses that are for sale across
the United States and finds that the mean price per square foot is 31.67
dollars, with a standard deviation of $1.29.

assume that the prices of
warehouse footage are normally distributed in population. If the researcher
uses a 5% level of significance, what statistical conclusion can be reached?
What are the hypotheses?


In [25]:
# Given
# sample = 19
# standard deviation = $1.29

# level of significance = 0.05 or 5%

# standard_error = standard_deviation/underoot of sample
# t_statistics = (sample_mean - population_mean)/standard_error

# Code:

# ho:32.28
# h1 != 32.28

standard_error = 1.29/np.sqrt(19)
print("standard error is {}%".format(standard_error*100))

t_statistic = (31.67 - 32.28)/standard_error
print("t_statistic is {}%".format(t_statistic))

print("-"*85)
print("So, my t_statistics value are not lie in the critical region that is -+2.101 , we failed to reject the null hypothesis")

print("-"*85)
print("He claim that is the average price per square foot for warehouses in the United States is 32.28 dollars is correct !!")


standard error is 29.594629669302464%
t_statistic is -2.06118477175179%
-------------------------------------------------------------------------------------
So, my t_statistics value are not lie in the critical region that is -+2.101 , we failed to reject the null hypothesis
-------------------------------------------------------------------------------------
He claim that is the average price per square foot for warehouses in the United States is 32.28 dollars is correct !!


# **Problem Statement 7**

Fill in the blank spaces in the table and draw your conclusions from it

# **Problem Statement 8:**


Find the t-score for a sample size of 16 taken from a population with mean

10 when the sample mean is 12 and the sample standard deviation is 1.5.

In [31]:
# sample = 16
# mean = 12 (sample_mean)
# mean = 10 (population_mean)
# standard_deviation = 1.5

standard_error = 1.5/np.sqrt(16)
t_statistic = (12-10)/standard_error

print("t-score is {}%".format(round(t_statistic,2)))

t-score is 5.33%


# **Problem Statement 9:**

Find the t-score below which we can expect 99% of sample means will
fall if

samples of size 16 are taken from a normally distributed population.


In [35]:
# sample = 16
# mean = 0.99 (sample_mean)


df = 15 # Degree of Freedom (16-1) see on row side of t_table
alpha = 0.01 # one tailed
print("t-score : 2.602")

t-score : 2.602


# **Problem Statement 10:**

If a random sample of size 25 drawn from a normal population gives a mean
of 60 and a standard deviation of 4,

 find the range of t-scores where we can
expect to find the middle 95% of all sample means. Compute the probability
that (−t0.05<𝑡<𝑡0.10).

In [6]:
#sample = 25
#sample_mean = 60
#standard deviation = 4


sample_mean = 60
sample_size = 25
sample_std = 4

std_err = sample_std / (sample_size)**0.5
print("The standard error is {}".format(std_err))

#t_score = (sample_mean - population_mean) / std_err

#For, df = 24 and alpha = 0.05 
# for a two tailed test because we need middle 95%:
#  t_critical = -2.064 to 2.064 (range)
print("Range of t-scores is from {} to {}".format(-2.064 , 2.064))
#Part b)

# Since, t0.1 leaves an area of 0.1 in the right and
#  -t0.05 leaves an area of 0.05 in the left. So, 
# P(−t0.05 < t < t0.10) = 1 - 0.1 - 0.05 = 0.85

The standard error is 0.8
Range of t-scores is from -2.064 to 2.064


# **Problem Statement 11:**

Two-tailed test for difference between two population means Is there
evidence to conclude that the number of people travelling from Bangalore
to Chennai is different from the number of people travelling from Bangalore
to Hosur in a week, given the following: 

Population 1: Bangalore to Chennai n1 = 1200

x1 = 452


s1 = 212

Population 2: Bangalore to Hosur n2 = 800

x2 = 523


s2 = 185

In [26]:
# Hypothesis Testing :
# H0: μ1 = μ2
# H1: μ1 ≠ μ2 

# As sample size is large, we will use z test:# As sample size is large, we will use z test:
# std_err = sample_std / (sample_size)**0.5
std_err = (((212)**2 / 1200) + ((185)**2) / 800)**0.5

z_statistic = (523 - 452) / std_err
print(z_statistic)
print("--"*30)
print("So, our z_statistic is too high or we can say it's p value is approximately zero.")
print("--"*30)
print("So, for any significance level, we can reject the null hypothesis !!")
print("--"*30)

7.926428526759299
------------------------------------------------------------
So, our z_statistic is too high or we can say it's p value is approximately zero.
------------------------------------------------------------
So, for any significance level, we can reject the null hypothesis !!
------------------------------------------------------------


# **Problem Statement 12:**

Is there evidence to conclude that the number of people preferring Duracell
battery is different from the number of people preferring Energizer battery,
given the following:


Population 1: Duracell

n1 = 100 x1 = 308 s1 = 84

Population 2: Energizer

n2 = 100 x2 = 254 s2 = 67


In [25]:
# Hypothesis Testing :
# H0: μ1 = μ2
# H1: μ1 ≠ μ2 

# As sample size is large, we will use z test:

std_err = (((84)**2 / 100) + ((67)**2 / 100))**0.5

z_statistic = (308 - 254) / std_err
print(z_statistic)
print("--"*30)
print("So, our z_statistic is too high or we can say it's p value is approximately zero.")
print("--"*30)
print("So, for any significance level, we can reject the null hypothesis i.e number of people using both are different")

5.025702668336442
------------------------------------------------------------
So, our z_statistic is too high or we can say it's p value is approximately zero.
------------------------------------------------------------
So, for any significance level, we can reject the null hypothesis i.e number of people using both are different


# **Problem Statement 13:**

Pooled estimate of the population variance. Does the data provide sufficient evidence to conclude that average percentage increase in the price of sugar differs when it is sold at two different prices?


Population 1: Price of sugar = Rs. 27.50 n1 = 14

x1 = 0.317% s1 = 0.12%

Population 2: Price of sugar = Rs. 20.00 n2 = 9

x2 = 0.21% s2 = 0.11%

In [39]:
# Hypothesis Testing :
# H0: avg1 = avg2
# H1: avg1 ≠ avg2 

# As sample size is small, we will use t test:

Sp = (((13* (0.12)**2) + (8 * (0.11)**2)) / 21)**0.5  # Sp = ((n1 * s1^2 + n2 * s2^2) / n1+n2-2) ^ 0.5

std_err = Sp * ((1/14) + (1/9))**0.5 # std_err = Sp * (1/n1 + 1/n2)**0.5

t_statistic = (0.317 - 0.21) / std_err
df = 21
# As t_statistic is coming 2.153. So, we will need level of significance in order to make a claim.

print("So, there's insufficient evidence to conclude anything !!")

#df = 21 and aplha = 0.05, t_critical = 2.08
print("But Null hypothesis can be rejected at 5% level of significance")

So, there's insufficient evidence to conclude anything !!
But Null hypothesis can be rejected at 5% level of significance


# **Problem Statement 14:**

The manufacturers of compact disk players want to test whether a small price reduction is enough to increase sales of their product. Is there evidence that the small price reduction is enough to increase sales of compact disk players?


Population 1: Before reduction

n1 = 15 

x1 = Rs. 6598

s1 = Rs. 844

Population 2: After reduction

n2 = 12 

x2 = RS. 6870 

s2 = Rs. 669

In [40]:
# Hypothesis Testing :
# H0: μ1 >= μ2
# H1: μ1 < μ2 

# As sample size is small, we will use t test:

Sp = (((14* (844)**2) + (11 * (669)**2)) / 25)**0.5  # Sp = ((n1 * s1^2 + n2 * s2^2) / n1+n2-2) ^ 0.5

std_err = Sp * ((1/15) + (1/12))**0.5 # std_err = Sp * (1/n1 + 1/n2)**0.5

t_statistic = (6870 - 6598) / std_err
print(t_statistic)
# As t_statistic is coming 0.91.

#df = 25 and even if we take alpha = 0.1 i.e 10% --> t_critical = 1.316

print("So, even at 10% level of significance, we cannot reject the null hypothesis!!")

0.9098300343990459
So, even at 10% level of significance, we cannot reject the null hypothesis!!


# **Problem Statement 15:**

Comparisons of two population proportions when the hypothesized difference is zero. Carry out a two-tailed test of the equality of banks’ share of the car loan market in 1980 and 1995.


Population 1: 1980

n1 = 1000

x1 = 53

p1 = 0.53

Population 2: 1985

n2 = 100

x2 = 43

p2= 0.43

In [42]:
# Hypothesis Testing :
# H0: p1 - p2 = 0
# H1: p1 - p2 ≠ 0 

# As sample size is large enough, we will use z test:

# p_cap = (x1 + x2) / (n1 + n2)
p_cap = (53+43) / (1000+100)
p_cap

std_err = np.sqrt(p_cap * (1 - p_cap) * (1/1000 + 1/100))

z_statistic = (0.53 - 0.43) / std_err

# As z_statistic is coming 1.415.

#even if we take alpha = 0.1 i.e 10% --> z_critical = 1.64

print("So, even at 10% level of significance, we cannot reject the null hypothesis!!")

So, even at 10% level of significance, we cannot reject the null hypothesis!!


# **Problem Statement 16:**

Carry out a one-tailed test to determine whether the population proportion of traveler’s check buyers who buy at least $2500 in checks when sweepstakes prize are offered as at least 10% higher than the proportion of such buyers when no sweepstakes are on.


Population 1: With sweepstakes

n1 = 300

x1 = 120 

p = 0.40

Population 2: No sweepstakes

n2 = 700

x2 = 140
p2 = 0.20

In [2]:
# Hypothesis Testing :
# H0: p1 - p2 <= 0.1
# H1: p1 - p2 > 0.1
# As sample size is large, we will use z test:
n1 = 300
x1 = 120
p1_cap = 0.40
n2 = 700
x2 = 140
p2_cap = 0.20
std_err = np.sqrt((p1_cap * (1 - p1_cap) / n1) + (p2_cap * (1 - p2_cap) / n2))
z_statistic = ((0.4 - 0.2) - 0.1)/ std_err
# As z_statistic is coming 3.11.
#even if we take alpha = 0.001 i.e 10% --> z_critical = 3.09
print("So, even at 0.1% level of significance, we can reject the null hypothesis!")

So, even at 0.1% level of significance, we can reject the null hypothesis!


**Chi-square test**

The chi-squared test is done to check if there is any difference between the observed value and expected value.

The formula for chi-square can be written as; or χ2 = ∑(Oi – Ei)2/Ei where Oi is the observed value and Eiis the expected value.

# **Problem Statement 17:**

A die is thrown 132 times with the following results:

Number turned up: 1, 2,
3, 4, 5, 6

Frequency: 16, 20, 25, 14, 29, 28

Is the die unbiased? Consider the degrees of freedom as p_cap -1 .

In [11]:

#H0 : The die is unbiased
#H1 : The die is biased

# Here, we will use the concept of chi-square test on categorical variables.
# chi_statistic = sum of ((Observed - Expected)^2 / Expected)
Expected = 132 * (1/6)
numbers = [1,2,3,4,5,6]
Observed = [16, 20, 25, 14, 29, 28]

chi_statistic_list = []

for i in Observed:
    c = (i-Expected)**2/Expected
    chi_statistic_list.append(c)
    


chi_statistics = sum(chi_statistic_list)

degree_of_freedom = len(numbers) - 1

#At 5% level of significance for 5 d.f. chi_critical = 11.07 which is > than chi_statistics

print("We cannot reject the null hypothesis. So, our die is unbiased")


We cannot reject the null hypothesis. So, our die is unbiased


# **Chi-Square Independence of test**

# **Problem Statement 18:**

In a certain town, there are about one million eligible voters. A simple
random sample of 10,000 eligible voters was chosen to study the
relationship between gender and participation in the last election.

The results are summarized in the following 2X2 (read two by two) contingency
table:









We would want to check whether being a man or a woman (columns) is independent of
having voted in the last election (rows). In other words, is “gender and voting
independent”?


In [9]:
#Step:1
# ho: gender and voting are independent
# h1: gender and voting are dependent


#Step:2
# Degree of freedom
# df = (row-1)(column-2)
# df = (2-1) (2-1)
# df = 1

#Step:3 (significance be 0.05 or 5%)
# crtical value = 3.841
#test of independence typically right tailed

#Step:4
#To Find expected value :
# E = (R_total)*(C_total)/N

#Expected Values:
#--------------------------
E_1_1 = (6383)*(4278)/10000
E_1_2 = (6383)*(5722)/10000
E_2_1 = (3617)*(4278)/10000
E_2_2 = (3617)*(5722)/10000
#---------------------------


#Step:5
#CHi-square test = sum((observed - expected)**2)/Expected)
chi_statistic = (2792-E_1_1)**2/E_1_1 + (3591-E_1_2)**2/E_1_2 + (1486-E_2_1)**2/E_2_1 + (2131-E_2_2)**2/E_2_2
print("chi_statistic is coming {} " .format(chi_statistic))



#As, chi_statistic is coming 6.66 which is greater than chi_critical (1 df and 5% sig. level) = 3.84

print("The hypothesis that the gender and voting are independent rejected at 5% level of significance.")

chi_statistic is coming 6.660455899328067 
The hypothesis that the gender and voting are independent rejected at 5% level of significance.


# **Chi-Squared Goodness-of-Fit test**

# **Problem Statement 19:**

A sample of 100 voters are asked which of four candidates they would vote for in an election.

The number supporting each candidate is given below:
Do the data suggest that all candidates are equally popular?


 [Chi-Square = 14.96, 
with 3 df, 𝑝<0.05 .

Higgin:41

Reardon:19

White:24

Charlton:16

In [10]:
#A Chi-Squared Goodness-of-Fit test is appropriate here. 
#H0: There is no preference for any of the candidates
#So, we would expect roughly equal numbers of voters to support each candidate. 
#Our expected frequencies are therefore 100/4 = 25 per candidate.

# chi_statistic = sum of ((Observed - Expected)^2 / Expected)


Expected = 25
Observed = [41,19,24,16]

chi_statistic_list = []

for i in Observed:
    chi_statistic_list.append((i-Expected)**2 / Expected)
    
chi_statistic = sum(chi_statistic_list) # value = 14.96

#The critical value of Chi-Square for a 0.05 significance level and 3 d.f. is 7.82.
#  Our obtained Chi-Square value is bigger than this.

print("so we conclude that voters do not prefer the four candidates equally.")

so we conclude that voters do not prefer the four candidates equally.


# Problem Statement-20

Children of three ages are asked to indicate their preference for three photographs of adults.

Do the data suggest that there is a significant relationship between age and photograph preference?

What is wrong with this study? [Chi-Square = 29.6, with 4 df: p < 0.05].

# Explanation Of Answer
A Chi-Squared Goodness-of-Fit test is appropriate here.

Firstly, lets calculate row, column
and grand totals.


Then, Work out the expected frequencies, using the formula:
 (row total*column total)
E = grand total chi_statistic = sum((O-E)^2 / E)


chi_statistic = 29.6 # After calculation



df = (columns-1) (rows-1) = 22 = 4


The critical value of Chi-Square in the table for a 0.05 significance level and 4 d.f. is 18.46.

Our obtained Chi-Square value is bigger than this: therefore we have a Chi-Square value which is so large that it would occur by chance only about once in a thousand times.

It
seems more reasonable to accept the alternative hypothesis, that there is a significant
relationship between age of child and photograph preference.

# **Problem Statement 21:**

A study of conformity using the Asch paradigm involved two conditions:

one where one confederate supported the true judgement and another where
no confederate gave the correct response

``Answer:``

A Chi-Squared Goodness-of-Fit test is appropriate here. Firstly, lets calculate row, column and grand totals.

Then, Work out the expected frequencies, using the formula:

 (row total*column total)
E = grand total chi_statistic = sum((O-E)^2 / E)

chi_statistic = 18.1 # After calculation df = (columns-1) (rows-1) = 1*1 = 1

Our obtained value of Chi-Squared is bigger than the critical value of Chi-Squared for a 0.001 significance level.

In other words, there is less than a one in a thousand chance of obtaining a Chi-Square value as big as our obtained one, merely by chance.

Therefore we can conclude that there is a significant difference between the "support" and "no support" conditions, in terms of the frequency with which individuals conformed.

# **Problem Statement 22:**

We want to test whether short people differ with respect to their leadership
qualities (Genghis Khan, Adolf Hitler and Napoleon were all stature-deprived, and how many midget MP's are there?)

 The following table shows
the frequencies with which 43 short people and 52 tall people were
categorized as "leaders", "followers" or as "unclassifiable".

 Is there a
relationship between height and leadership qualities? [Chi-Square = 10.71,
with 2 df: p < 0.01].

**Answer**
``Chi-square goodness of fit``

- chi_statistic = 10.712 with 2 d.f.

- 10.712 is bigger than the tabulated value of Chi-Square
- at the 0.01 significance level. 

- We would conclude that there seems to be a relationship between height and leadership qualities.

# **Problem Statement 23:**

Each respondent in the Current Population Survey of March 1993 was
classified as employed, unemployed, or outside the labor force. The results
for men in California age 35- 44 can be cross-tabulated by marital status, as
follows:


Men of different marital status seem to have different distributions of labor force status.

 Or is this just chance variation? (you may assume the table results from a simple random
sample.)


``Formula``
x2 = sum  or  sigma(observed - expected )/expected 

chi_statistic is coming very large as

compared to the values for any significance level at df
= 4.

So, it can be concluded that yes,

 Men of different marital status seem to have different
distributions of labor force status.


**Completed**